src/input.service.ts

```diff
@@ -264,8 +272,16 @@ function isForForm(direction: Direction, selected: Element): boolean {
 @Injectable()
 export class InputService {
 
-  private inputPane = Windows.UI.ViewManagement.InputPane.getForCurrentView();
-  public keyboardVisible = new BehaviorSubject(false);
+  /**
+   * Inputpane and boolean to indicate whether it's visible
+   */
+  private inputPane = (<any>window).Windows ? Windows.UI.ViewManagement.InputPane.getForCurrentView() : null;
+  public keyboardVisible = (<any>window).Windows ? new BehaviorSubject(false) : null;
```

Comment:
Would it make sense to keep it as a `new BehaviorSubject(false)` even when we aren't in a windows environment? Seems like a sensible default and streamlines other code.